module/userProjects/helper.js

```diff
@@ -940,7 +993,7 @@ module.exports = class UserProjectsHelper {
                 );
                 if (!project.length > 0) {
                     throw {
-                        status: HTTP_STATUS_CODE['bad_request'].status,
+                        status: HTTP_STATUS_CODE.bad_request.status,
```

Comment:
@Prajwal17Tunerlabs modify the _observationDetails function you dont need to call await surveyService.createObservationFromSolutionTemplate( make change

Reply:
we are not using this function